client/.prettierrc.toml

```diff
@@ -0,0 +1,10 @@
+printWidth = 90
+semi = false
+singleQuote = false
+quoteProps = "preserve"
+trailingComma = "none"
+bracketSpacing = true
+arrowParens = "avoid"
+# let git handle normalizing
+endOfLine = "auto"
+embeddedLanguageFormatting = "off"
```

Comment:
No, it's just annoying to handle. Git automatically converts LF into CRLF for Windows, and CRLF to LF when committing. I don't want to do the tricks required to override this behavior, and it's a very normal feature in Git, so it's not brittle.

Reply:
Sorry, I was referring specifically to `embeddedLanguageFormatting `

Reply:
Oh, well, it would be inconsistent. If you had some embedded `HTML` in some JS, and had this on, it would format the `HTML`, but we don't format `HTML`.

Reply:
Good point